pwiz_tools/Shared/CommonMsData/RemoteApi/WatersConnect/WatersConnectSession.cs

```diff
@@ -18,15 +18,24 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
 using Newtonsoft.Json.Linq;
 using pwiz.Common.Collections;
 
 namespace pwiz.CommonMsData.RemoteApi.WatersConnect
 {
     public class WatersConnectSession : RemoteSession
     {
+        public static readonly string GET_FOLDERS = @"/waters_connect/v1.0/folders";
+        protected HttpClient _httpClient;
+
         public WatersConnectSession(WatersConnectAccount account) : base(account)
         {
+            if (account == null)
+                throw new ArgumentException(@"WatersConnectSession requires a WatersConnectAccount");
```

Comment:
Would we expect a user to see this? If so, resx. If not, I think we use Assume for this, right @nickshulman ?